src/client/store/ui/assessmentFiles/actions/updatePublic.ts

```diff
@@ -0,0 +1,26 @@
+import { createAsyncThunk } from '@reduxjs/toolkit'
+import axios from 'axios'
+
+import { ApiEndPoint } from 'meta/api/endpoint'
+import { CycleParams } from 'meta/api/request'
+import { CountryIso } from 'meta/area'
+import { AssessmentFile } from 'meta/cycleData'
+
+type Params = CycleParams & {
+  fileCountryIso?: CountryIso
+  files: Array<AssessmentFile>
+  public: boolean
+}
+
+export const updatePublic = createAsyncThunk<Array<AssessmentFile>, Params>(
```

Comment:
repository left as `updateMany` because we are not updating only access, but anything that is passed in rows. We can use this for cycle updates later 👍